kubernetes/polkassembly/values.yaml

```diff
@@ -151,3 +151,39 @@ chainDbWatcher:
   resources: {}
   tolerations: []
   affinity: {}
+
+healthMonitor:
+  name: health-monitor
+  replicaCount: 1
+
+  config:
+    # Listening Port
+    PORT: 8034
+    # polkassembly auth-server
+    AUTH_SERVER: "http://auth-server-service:8010"
+    # polkassembly hasura graphql service
+    HASURA_SERVER: "http://hasura-service:8080"
+    # polkassembly hasura chain-db service
+    CHAIN_DB_WATCHER_SERVER: "http://chain-db-watcher-service:8019"
+    # nomidot backend server ( CHAIN_DB_SERVER == CHAIN_DB_GRAPHQL_URL )
+    CHAIN_DB_SERVER: "http://nomidot-server.nomidot-staging:4000"
```

Comment:
Same here, isn't `http://35.189.196.74:4000` safer because the domain would only be accessible from the same `staging` container, whereas the IP works from outside as well?
```suggestion
    CHAIN_DB_SERVER: "http://35.189.196.74:4000"
```

Reply:
`https://test.nodewatcher-server.polkassembly.io` is what we want to request here i guess, right? it's the ingress pointing to `nodewatcher-server:4000` in staging. 
but why is the variable then named `CHAIN_DB_SERVER`?

Reply:
> https://test.nodewatcher-server.polkassembly.io is what we want to request here i guess, right?

Yes that sounds right. `CHAIN_DB_SERVER` because it's the server we reach from hasura to get chain-db data. It's the server that is open to the world and only has read access to the DB.